streaming/index.js

```diff
@@ -148,14 +132,19 @@ const pgConfigFromEnv = (env) => {
  */
 const redisConfigFromEnv = (env) => {
   const redisNamespace = env.REDIS_NAMESPACE || null;
+  let path = null;
+  if (env.REDIS_URL.startsWith('unix://')) {
+    path = env.REDIS_URL.slice(7);
+  }
 
   const redisParams = {
-    socket: {
-      host: env.REDIS_HOST || '127.0.0.1',
-      port: env.REDIS_PORT || 6379,
-    },
-    database: env.REDIS_DB || 0,
-    password: env.REDIS_PASSWORD || undefined,
+    host: env.REDIS_HOST || '127.0.0.1',
+    port: env.REDIS_PORT || 6379,
+    options: {
+	  path: path,
+	  db: env.REDIS_DB || 0,
+      password: env.REDIS_PASSWORD || undefined,
+	}
```

Comment:
Indentation here is a bit wonky, be sure to check this with prettier/linting (`npm run lint` from the root, or `prettier --check streaming/index.js`)

```suggestion
  	  path: path,
  	  db: env.REDIS_DB || 0,
      password: env.REDIS_PASSWORD || undefined,
	  }
```